borg/helpers.py

```diff
@@ -124,8 +129,22 @@ def check_extension_modules():
         raise ExtensionModuleError
 
 
+SUPPORTED_REPO_FEATURES = frozenset()
```

Comment:
minor; move this constant to Manifest as well, where the other related constants live as well?